src/components/submission-common.js

```diff
@@ -12,6 +12,8 @@ import colors from 'src/libs/colors'
 import * as Utils from 'src/libs/utils'
 
 
+export const AutoRefreshTimeout = 1000 * 60 // 1 minute
```

Comment:
For reviewers: What should be the timeout? This value is same as we have in Terra UI. I can increase it if we think this will put too much load on CBAS.

Reply:
My first instinct is to do whatever is done in Terra UI. Curious to see what others think as well.

Reply:
agreed, "same as terra UI" is a good starting point until we have a reason to change it. 

it actually feels kind of long to me, from a user perspective, but that's a conversation for another time